### ISSUE TITLE
Components and headings

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#301]
## What
The new 2i checklist for NHS content designers says: 
"Check components with a heading (such as a warning callout) do not have content below them, as this is an accessibility issue. They should be followed by a new heading."

We'd like to get something into relevant service manual components about this but could do with being clearer:

- which components it applies to (care cards, warning callouts, do and don't lists? Do we ever use headings with inset text?)
- how best to describe the accessibility issue

## Why
If this is best practice, we want to be sure that all content designers understand the guidance and implement it and that our internal processes reflect the service manual and vice versa.

**Comment by sarawilcox** [comment]
Pic 1 is what a sighted person sees (only the two sentences in the yellow box are "important"). Pic 2 is what a screenreader user would "see" (the two sentences and the bullets below are both "important").
![pic1](https://user-images.githubusercontent.com/45792959/103911664-3372aa00-50fe-11eb-9f15-8e21ab23188a.png)
![pic2](https://user-images.githubusercontent.com/45792959/103911717-438a8980-50fe-11eb-9e3b-5df5a473c772.png)

**Comment by sarawilcox** [comment]
After a brief catch up about this:
- @mcheung-nhs will double-check that it is an issue across various screen readers
- @mcheung-nhs and @davidhunter08 to check out whether there's any way to define the end of the component so that users know that the heading doesn't apply to content that follows the component

Assuming that this is an accessibility issue and there isn't a code-based solution, @sarawilcox will draft updates for the following components:
- care cards
- warning callouts
- inset text
- do and don't lists

We need to explain that there shouldn't be any text between a component which has a heading and the next heading. This is because there's no indication for screen reader users where the component ends, so any content that follows the component will look like it's part of it. 

To be approved at a Style Council meeting - probably February 2021.

**Comment by mcheung-nhs** [comment]
I've checked VoiceOver and NVDA and JAWS and there is no indication that you have exited a care card, warning callout or inset text. One solution to indicate that you have come to the end of a component is to have hidden text for screen readers to say so. However, I'm not sure if this is a good solution so would recommend doing some user research to see if it effective or required.
So, in the meantime I would go with the recommendations of not having any text between one of the components which has a heading and the next heading.

**Comment by sarawilcox** [comment]
Thanks @mcheung-nhs. We could take that recommendation to the Feb Style Council meeting.

Alistair is recommending that we consider doing some research around content and accessibility. Perhaps we can discuss the following at our meeting with him on 3 Feb:
- this issue
- the [inset text issue](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/18)
- the [pronunciation issue](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/298)
- [hint text}(https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/17)

Are there more?

@amyj2110 and @davidhunter08, would you be interested in joining the meeting and exploring how we might test issues around headings, punctuation, components and accessibility?

**Comment by sarawilcox** [comment]
We discussed this issue with Alistair Duggin and then at a follow up meeting today. 

- We may be able to find a way of announcing not only the start but also the end of the component.
- For the time being, content designers are being advised to avoid hanging text directly under a component without a heading. 
- We're not proposing to make any changes to our approach until we've had a chance to test things with screen reader users. We don't know to what extent this does represent a problem to them. @davidhunter08 hadn't seen any problems in testing with a blind user.
- @KarinMochan and @sarawilcox will look out some examples to test with users, including care cards. @markasrobinson, anything you'd recommend that we test?
- @mcheung-nhs and @davidhunter08 will prepare a prototype with examples to test with users.

**Comment by sarawilcox** [comment]
An example from our [aciclovir ](https://www.nhs.uk/medicines/aciclovir/) page. Text including inset component between a care card and the next heading.
![Uploading Screenshot 2021-02-18 at 11.46.24.png…]()

**Comment by sarawilcox** [comment]
An example from [low dose aspirin](https://www.nhs.uk/medicines/low-dose-aspirin/). Text between a warning callout and the next heading. 
![Screenshot 2021-02-18 at 11 52 32](https://user-images.githubusercontent.com/45792959/108353441-0f67b580-71e0-11eb-8382-56186451e490.png)

**Comment by sarawilcox** [comment]
This is one of several accessibility themed content issues that we need to test in a prototype.
